samples/bot-teams-catering/csharp/Manifest/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.7/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.7",
+    "version": "2.0.0",
+    "id": "be518d02-7ebc-4f26-8f72-284aa8a43349",
```

Comment:
Could you pleaes add placeholder ( <<App ID>>) instead of App Id?

Reply:
done